extensions/interactions/CodeRepl/CodeRepl.js

```diff
@@ -119,7 +119,7 @@ oppia.directive('oppiaInteractiveCodeRepl', [
           execLimit: 10000
         });
 
-        $scope.runCode = function(codeInput) {
+        $scope.runAndSubmitCode = function(codeInput, status) {
```

Comment:
I thought the idea was to have two functions? runAndSubmitCode() would call runCode() with one argument, 'onFinishEvaluationCallback'.